source/deployment/bleve.rst

```diff
@@ -20,10 +20,15 @@ Follow these steps to configure the Mattermost server to use Bleve and generate
 6. Set **Enable Bleve for search queries** to **true**.
 7. Set **Enable Bleve for autocomplete queries** to **true**.
 
+.. note::
+
```

Comment:
I'd remove this line break for consistency - although I tend to use a line break myself, so maybe adding a line break to the other notes is better? I don't think we have a set standard for this - 0/5 for either.